compiler/dialects/stream.py

```diff
@@ -128,7 +128,7 @@ class GenericOp(IRDLOperation):
 
     name = "stream.generic"
 
-    inputs = var_operand_def(StreamType)
+    inputs = var_operand_def()
```

Comment:
generics can also have i32 inputs, but it's rabbit hole in python typing hell to make this work without angering pyright, so just like in the regular linalg.generic there is no constraint

Reply:
Maybe put this as a comment then :)